src/main/java/com/graphhopper/converter/resources/ConverterResourceOpenCageData.java

```diff
@@ -41,17 +41,19 @@ public Response handle(@QueryParam("q") @DefaultValue("") String query,
                            @QueryParam("nominatim") @DefaultValue("false") boolean nominatim,
                            @QueryParam("find_osm_id") @DefaultValue("true") boolean find_osm_id,
                            @QueryParam("reverse") @DefaultValue("false") boolean reverse,
-                           @QueryParam("point") @DefaultValue("false") String point
+                           @QueryParam("point") @DefaultValue("false") String point,
+                           @QueryParam("ocd_key") @DefaultValue("") String ocdKey
```

Comment:
Allowing to provide an ocd_key per request seemed the cleanest solution to inject the key from an environment variable. Another solution could have been to check if there is an environment variable in `ConverterApplication`. Let me know if I should move the code there. For now I decided against it, because it's too much "magic" for my taste. When debugging an issue you would easily miss this and wonder why it works locally but not on the server.

Reply:
Will this work?

https://stackoverflow.com/a/32231630/194609
https://www.dropwizard.io/1.3.0/docs/manual/core.html#environment-variables

Reply:
Ah, good point! This works great. BTW: We could also use the same approach for Gisgraphy and specify private environment variables for this as well?

Reply:
ok, sure

Reply:
can I remove the user-agent that allow requests (for tests)  on https://github.com/graphhopper/geocoder-converter/issues/48 ?

If I understand Travis and other will use an env variable as the API key, right ?

Reply:
> If I understand Travis and other will use an env variable as the API key, right ?

Yes, that is correct. I think you can remove the user agent :).